packages/cf-component-icon/src/Icon.js

```diff
@@ -76,11 +288,18 @@ Icon.propTypes = {
     'widen',
     'wrench'
   ]).isRequired,
-  size: PropTypes.oneOf(['2x', '3x', '4x', 'large', 'xlarge']),
```

Comment:
These styles came out of the existing backbone code base, they could be useful at some point. If it's not much effort, I think it makes sense to leave that in.